src/app/postulants/postulants.component.html

```diff
@@ -60,17 +60,14 @@ <h3>Postulants</h3>
                 postulant.phone
               }}</a>
             </td>
-            <td class="center-align">
```

Comment:
Put this field back. You are supposed to remove the one with `postulant.bevyFilled` binding, not the one with `postulant.validated` binding